src/AppInstallerCommonCore/Public/winget/JsonUtil.h

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include <json.h>
+
+#include <optional>
+#include <string>
+
+namespace AppInstaller::Utility
```

Comment:
Why does this file need to be public?  Probably fine for it to be for potential future use, but it shouldn't be need right now.